Feature/test branch script

jhagberg opened the pull request:
Do not know if this will be of any help but it speeds up testing branches that have prebuilt images !

Commit:
Script to test the current branch with remote prebuilt images

Added a script to get the current branch as image tag same
way as the githubaction tags the images

Added a docker compose file that use the remote images

Commit:
Added info of the test branch script

norling commented on README.md:
````diff
@@ -68,3 +68,9 @@ To be able to login in the website and play with it you will need to create an u
 ```
 
+If you just want to test a PR or a branche that is pushed to Github we now have a Github Action that will build the images
+So to just pull and test the prebuilt images use the script
````
misspelled "branch". Also - I think it might be clearer written as:
```
All branches that are pushed to github has prebuilt images. To use the prebuilt images instead of building locally, use:
```
or something like that :)

jhagberg replied:
Make a suggestion and commit! :)

norling commented on dc-current-branch.yml:
```diff
@@ -0,0 +1,43 @@
+        volumes:
+            - ./postgres-data:/var/lib/postgresql/data
+    herdbook-frontend-devel:
+        image: ghcr.io/nbisweden/herdbook_frontend:${TAG}
```
It might be neat to use the `${TAG:?error-message}`-syntax to give a helpful error message if TAG isn't set properly!

norling commented on test-current-branch.sh:
```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+GITHUB_REF="$(git rev-parse --abbrev-ref HEAD)" TAG=${GITHUB_REF##*/} docker-compose -f dc-current-branch.yml up
```
newline isn't a more expensive character than space.

Commit:
Update README and rename of script

Commit:
Added a error message if TAG is not set or empty

Commit:
Export TAG

Export the TAG var so it works on other bash versions

Commit:
UPDATE readme missed to add

Commit:
Do a separate pull to get the latest images

Commit:
New compose file that use prod env

Made a new compose file that is more production like
This ensures we test the brebuilt images and not any local changes

pontus commented:
Could possibly add the option of passing a branch to run for instead of the current, but this is very good as is.

jhagberg commented:
Yes we talked about that it could be good but we can just do an new PR! :)
